.github/workflows/release-internal.yml

```diff
@@ -32,20 +32,56 @@ on:
         required: true
         default: false
 
+  schedule:
+    - cron: '0 13 * * 3' # Every Wednesday at 8:00 AM central time
+
 defaults:
   run:
     shell: bash
 
 jobs:
+  get-last-successful-release-run:
+    runs-on: ubuntu-latest
+    outputs:
+      commit_sha: ${{ steps.get_last_successful_release_run.outputs.commit_sha }}
+    steps:
+      - name: Get last successful Release to Cloud run details
+        id: get_last_successful_release_run
+        run: |
+          last_run_url=$(curl --silent -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" "https://api.github.com/repos/${{ github.repository }}/actions/runs?event=workflow_dispatch&status=success&branch=${{ github.ref }}&workflow_file=release.internal.yml" | jq -r '.workflow_runs[0].url')
+          echo "Last run URL: $last_run_url"
+          commit_sha=$(curl --silent -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" $last_run_url | jq -r '.head_sha')
+          echo "Commit SHA: $commit_sha"
+          echo "::set-output name=commit_sha::$commit_sha"
```

Comment:
I think this needs to be the new syntax that uses `>> GITHUB_OUTPUT`. I forget what it is off the top of my head, but it's something like `echo "set commit_sha=$commit_sha" >> GITHUB_OUTPUT`.